### ISSUE TITLE
Support for java.lang.SuppressWarnings

### ISSUE THREAD

**Issue by juddgaddie** [spotbugs/spotbugs#840]
Supporting java.lang.SuppressWarnings would be that modules would not require to hava a compile time dependency on Spotbugs annotations - it could be in the form @SuppressWarnings({"spotbugs:KEY"})

I am  happy to submit a patch if its something that would be accepted.

related to #737

**Comment by jjlharrison** [comment]
I don't think this is possible because the `SuppressWarnings` annotation has source retention and is therefore not visible to SpotBugs.

**Comment by KengoTODA** [comment]
Reference flr future use: Spotbugs has similar annotation.

https://static.javadoc.io/com.github.spotbugs/spotbugs-annotations/3.1.10/edu/umd/cs/findbugs/annotations/SuppressFBWarnings.html
